src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+xref: OMIA:002052 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002052"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0000328 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog hyperphosphatemia
+
+[Term]
+id: MONDO:1010111
+name: tyrosinemia type I, non-human animal
+def: "Tyrosinemia type I that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002078 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002078"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002078"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010161 {source="OMIA:002078"} ! cross-species analog tyrosinemia type I
+
+[Term]
+id: MONDO:1010112
+name: vitamin D-deficiency rickets type 1B, non-human animal
+def: "Vitamin d-deficiency rickets type Ib that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002221 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002221"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002221"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010810 {source="OMIA:002221"} ! cross-species analog vitamin D hydroxylation-deficient rickets, type 1B
+
+[Term]
+id: MONDO:1010113
+name: glucocorticoid resistance, non-human animal
+def: "Glucocorticoid resistance that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002254 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002254"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002254"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0014421 {source="OMIA:002254"} ! cross-species analog glucocorticoid resistance
+
+[Term]
+id: MONDO:1010114
+name: phenylketonuria, non-human animal
+def: "Phenylketonuria that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002293 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002293"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:002293"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009861 {source="OMIA:002293"} ! cross-species analog phenylketonuria
+
+[Term]
+id: MONDO:1010115
+name: c8 deficiency, non-human animal
+def: "C8 deficiency that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000156 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000156"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000156"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0013422 {source="OMIA:000156"} ! cross-species analog type I complement component 8 deficiency
+
+[Term]
+id: MONDO:1010116
+name: leukocyte adhesion deficiency type I, non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.